server/src/main/java/com/objectcomputing/checkins/services/guild/Guild.java

```diff
@@ -33,6 +36,12 @@
     @Schema(description = "name of the guild")
     private String name;
 
+
+    @Nullable
+    @Column(name="link", unique=true)
+    @Schema(description="link to OCI compass page of guild")
+    private String link;
```

Comment:
You're going to hate me, but I think I would encrypt this field. It can provide hints that indicate persona to someone who gets into the database. It's a small risk, but it's there.

Reply:
I encrypted the links.